src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
```

Comment:
Please remove the redundant empty line after the class declaration to adhere to the coding standards mentioned in the checklist.